packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -718,6 +742,13 @@ export default async function initSanity(
     )
     print('\nGet started in `src/App.tsx`, or refer to our documentation for a walkthrough:')
     print(chalk.blue.underline('https://www.sanity.io/docs/app-sdk/sdk-configuration'))
+    if (mcpConfigured && mcpConfigured.length > 0) {
+      const editorNames = mcpConfigured.map((e) => e.name).join(' and ')
```

Comment:
See rationale on same thing above
```suggestion
      const editorNames = new Intl.ListFormat('en').format(mcpConfigured.map((e) => e.name))
```